GeneratingDataset.py

```diff
@@ -1316,18 +1316,20 @@ def create_vocab(cls, **opts):
       clz = BytePairEncoding
     return clz(**opts)
 
-  def __init__(self, vocab_file, unknown_label="UNK", num_labels=None):
+  def __init__(self, vocab_file, unknown_label="UNK", num_labels=None, seq_postfix=[]):
```

Comment:
If you add it here, then you should delete it in `BytePairEncoding`. Also, never use a list (`[]`) as default param (see PyCharm comment why that is a bad idea). Anyway, use the same default param as in `BytePairEncoding`. Basically, just make it exactly the same as in `BytePairEncoding`.